src/emrun_prejs.js

```diff
@@ -1,7 +1,8 @@
-// Copyright 2013 The Emscripten Authors.  All rights reserved.
-// Emscripten is available under two separate licenses, the MIT license and the
-// University of Illinois/NCSA Open Source License.  Both these licenses can be
-// found in the LICENSE file.
+/**
```

Comment:
Why the double `*` , is that part of the format/requirement

Reply:
I think it might be, I copied it directly from the docs I was pointed to for proper usage of SPDX. I'd rather not many risky alterations.

Reply:
The double star format is normally accosted with doxygen, so I'm a little surprised to see it here.  Or maybe this is part of JSDoc somehow which uses the same convention?

Reply:
I'm really not sure. I sent you a link to the docs I was given.

Reply:
Yes, it looks like its JSDoc format which explains this.